src/js/tooltip.js

```diff
@@ -60,12 +94,28 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         }
     });
 
-    gpii.firstDiscovery.attachTooltip.getTooltipModel = function (domBinder, msgResolver, map) {
+    // Return an object the element id and text for the given label
+    gpii.firstDiscovery.attachTooltip.getElementInfo = function (msgResolver, element, label) {
+        var info = {};
+        info.id = fluid.allocateSimpleId(element);
+        info.msg = msgResolver.resolve(label);
+        return info;
+    };
+
+    gpii.firstDiscovery.attachTooltip.getTooltipModel = function (domBinder, map, getElementInfo, currentSelectedIndex) {
         var idToContent = {};
 
-        fluid.each(map, function (string, selector) {
-            var id = fluid.allocateSimpleId(domBinder.locate(selector));
-            idToContent[id] = msgResolver.resolve(string);
+        fluid.each(map, function (labelInfo, selector) {
+            var element = domBinder.locate(selector);
+            if (typeof (labelInfo) === "object") {
```

Comment:
typeof "object" is not particularly reliable in JavaScript - I suggest the use of fluid.isPrimitive(), or else a test against "string" if that is the expected type for getElementInfo